app/src/main/java/io/github/joshkergan/giftr/db/MappingContract.java

```diff
@@ -1,25 +1,25 @@
 package io.github.joshkergan.giftr.db;
 
 import android.provider.BaseColumns;
-import android.provider.ContactsContract;
 
 /**
  * Created by Patrick on 16/10/2016.
  * Contract for the Mapping Table, which maps the Item table to the People table
  */
 
 public final class MappingContract{
-    // No external instantiation
-    private MappingContract() {}
+	// No external instantiation
+	private MappingContract() {

```

Comment:
We explicitly wanted this fixed in a previous commit, which is why I'm enforcing consistency here.